docs: Chinese translation for Mental Health in Open Source (#74)
antfu/antfu.me: pages/posts/mental-health-oss-zh.md

- {@KermanX} {@tonai} 在 {Slidev|https://github.com/slidevjs/slidev} 上推动了很多功能
- {@arashsheyda} 在 {Nuxt DevTools|https://github.com/nuxt/devtools} 上提供了很多帮助
- {@shuuji3} {@Shinigami92} 为 {Elk|https://github.com/elk-zone/elk} 做出了贡献
- {@patak-dev} {@sapphi-red} {@bluwy} 与出色的社区一起推动 Vite 前进
- {@userquin} 维护 {Vite PWA|https://github.com/vite-pwa} 并在各个地方提供帮助
- {@yyx990803}，我从他那学到了很多关于开源和决策的东西
- ...以及许多为开源做出贡献或通过赞助提供财务支持的人！

很遗憾我无法列出所有的人，而且他们中的许多人实际上在项目之间是重叠的。我想说的是，我不是一个人在工作，我不能一个人做所有这些。我从社区和团队那里**得到了很多帮助**来完成所有这些项目。我对此深表感激。除了<b important-text-hex-80BEDF>质量</b>和<b important-text-hex-E3B65E>速度</b>之外，似乎我还在广泛<b important-text-hex-D777B1>范围</b>的项目上工作 —— 看起来好像打破了铁三角的规则 —— 但实际上，幕后的出色社区是使这一切成为可能的“魔法”。

### 容量

<!-- TODO: 容量感觉不太好，但找不到词语替代 -->

维护多个高流量的开源项目所需的工作量实际上是巨大的。没有社区的帮助，我早就应该达到了我的极限。尽管社区给了我很多帮助，但是，沟通、协调以及一贯的上下文切换仍然需要很多精力。随着时间的推移，我积累了很多我必须自己做的事情，我想尝试的很多想法，以及我想改进的很多东西。

我想让这些项目保持活力并继续前进；我想写更多的博客文章来分享我的想法；我想做更多的演讲，去旅行，见见人；我想做更多的直播，因为我知道很多人在等着；我必须整理，并发布；我也想学法语；并花更多的时间和家人在一起 —— 我的意思是，这可能只是生活。人们有自己的担忧和责任，我并不比其他人更特别或更忙。

> <span font-wisper font-bold op80 text-1.4rem leading-2.1rem>“但不知怎么回事，有些东西让我很压抑。”</span>

<!-- TODO: 这里去掉了换行，个人对字体没什么研究，可能有必要调整下~ -->

> 我可能不愿意承认我潜在的疲劳。不是因为我害怕，而是更像是，我不想放弃，被动地处理它。我知道需要休息时要休息，但称自己为“疲劳”并放弃是一种“逃避责任”的“捷径”。我想找出“根本原因”，并尝试改善情况，而不是只是“绕过”它。正如我们之前讨论的，对“期望”的转变，以及对我“准备不足”和“自我期望”的重新评估是我在不同原因下接近疲劳时的解决方案。通过调整自己和接纳现实，我通常能在大约一周的时间内从低谷中恢复过来，继续前进。

Reviewer: ```suggestion
我可能不愿意承认我潜在的倦怠。这并不是因为我害怕，而是我不想轻言放弃，也不想被动地去应对。我知道需要休息时要休息，但称自己为“倦怠”并放弃对我来说像是一种“逃避责任”的“捷径”。我想找出“根本原因”，并尝试改善情况，而不仅仅是「临时应对」。正如我们之前讨论的，对“期望”的转变，重新审视自己的“措手不及”和“自我期望”，是我在因不同原因而面临倦怠时的解决方案。通过自我调整和适应，我通常能在大约一周的时间内走出低谷，并继续前进。
```

> 这次的情况有点不同，不是因为我没有动力，而是因为我想做的事情太多了，但我已经没有了容量。我开始思考，也许是我期望自己保持以相同的<b important-text-hex-E3B65E>速度</b>交付一切，我对做得不够和做得不快感到焦虑。快速获得反馈是很棒的，也很高效，但我可能因为太习惯于快速而变得有些不耐烦。结合起来，当我在做一些需要中长期努力的事情时，我很容易感到沮丧。

Reviewer: ```suggestion
这次的情况有点不同，并不是因为我缺乏动力，而是因为我想做的事情太多了，而是因为我想做的事情太多，超出了我的承受能力。我开始思考，或许问题在于我总是抱着保持<b important-text-hex-E3B65E>速度</b>并且不断交付成果这种期望，我对做得不够和做得不快感到焦虑。快速获得反馈是很棒的，也很高效，但我可能因为太习惯于快速而变得有些不耐烦。这些因素叠加在一起，当我在做一些需要中长期努力的事情时，我很容易感到沮丧。
```

> 例如，_写作_。我不擅长写作，而且我并不真的喜欢它。文档、博客文章、教程和演讲 —— 都需要很多时间，也是我必须做的事情。当我在写作时，我很容易分心和失去专注，或者在中途放弃。所以我在 [Twitter 上询问了一下](https://twitter.com/antfu7/status/1764397930796953823)，得到了社区的很多很好的建议（看看评论，你可能也会找到对你有用的东西）。我开始尝试放轻松，慢慢地做，试图转变我的思维，不期望立即的结果，享受过程。你正在阅读的这篇博客大约花了我一周的时间来写（对我来说这是相当长的时间），分成了多个会话。我觉得这也帮助我重新思考和整理我的思想和感受，实际上，把它们写下来让我对我的焦虑感觉好多了。

Reviewer: ```suggestion
例如，_写作_。我不擅长写作，而且我也并不喜欢写作。文档、博客文章、教程和演讲 —— 都需要耗费大量时间，但我又不得不做。当我在写作时，我很容易失心和分身，或者中途放弃。所以我在 [Twitter 上询问了一下](https://twitter.com/antfu7/status/1764397930796953823)，得到了社区的很多很好的建议（看看评论，对你来说也许也有帮助）。我开始尝试放轻松，慢慢地做，试图转变我的思维，不急于求成，享受过程。你正在阅读的这篇博客大约花了我一周的时间来写（对我来说这是相当长的时间），分成了多个阶段来完成。这也帮助我重新审视和梳理我的思想和感受，实际上，把它们写下来也减轻了我的焦虑感。
```

> 所以，我可能应该重新评估我的容量和期望。我必须理解并接受，我并不总是能保持相同的速度，也不必逼自己太紧。放慢一点，更多地关注细节，也许我会在过程中找到不同的快乐和满足感。

Reviewer: ```suggestion
因此，我应该重新审视一下自己的承受力和期望值。我得明白并接受一个事实：我不可能始终以这样的速度前进，也没必要对自己过于严苛。放缓步伐，更细致地打磨每一个细节，也许我能在这个过程中找到别样的快乐和满足感。
```

> 老实说，我甚至不确定我在这篇博客文章中想要表达什么 —— 也许只是简单地与你分享我的想法和感受。现在，我仍然感到相当大的压力。我仍在适应和努力寻找更好的处理方式。在这一周的写作和与朋友交谈中，我感觉好多了，我相信我会度过这个难关。这可能就像我们生活中的许多其他事情一样；我们并不总是有完美的解决方案，但我们必须继续前进，找到自己的出路。

Reviewer: ```suggestion
老实说，我并不确定我在这篇博客文章中想要表达什么 —— 也许只是简单地与你分享我的想法和感受。现在，我仍然感到相当大的压力。我仍在适应和努力寻找更好的处理方式。通过这一周的写作和与朋友交谈中，我感觉好多了，我相信我会度过这个难关。这可能就像我们生活中的许多其他事情一样；我们并不总是有完美的解决方案，但我们必须得继续前进，找到出路。
```

> 保持良好的心理健康是每个开源维护者保持可持续性的重要任务之一。我不认为在整个旅程中会有“一个答案”或“一个解决方案”。这更像是一个持续的学习和适应的过程，找到适合我们每个人的方法。

Reviewer: ```suggestion
对于每个开源项目维护者来说，保持良好的心理健康是实现长期可持续发展的关键。在整个旅程中，我不认为会有一个统一的“答案”或“解决方案”。这更像是一个持续的学习和适应的过程，去寻找适合我们每个人的方法。
```

### 我很想听到你的想法

> **谢谢**你读完了我混乱而冗长的心情直到最后！

Reviewer: ```suggestion
**感谢**你能耐心地把我这杂乱冗长的心情读到最后！
```

> 我知道我的观点一定是非常偏见的。如果它曾经引发了你的一些想法或感受，我**很想听听你的想法或你的方法**。你可以在这个 [推文](https://twitter.com/antfu7/status/1768973757681438909) 或 [mastodon](https://elk.zone/m.webtoo.ls/@antfu/112105304547243075) 下留下一些评论，或者如果你更喜欢私人对话，可以给我发邮件

Reviewer: ```suggestion
我知道我的观点一定是非常偏见的。如果它曾经引发了你的一些想法或感受，我**很想听听你的想法或你的方法**。你可以在这个 [推文](https://twitter.com/antfu7/status/1768973757681438909) 或 [mastodon](https://elk.zone/m.webtoo.ls/@antfu/112105304547243075) 下评论，如果你更喜欢私下交流，也可以发邮件到 hi@antfu.me。
```

> 与此同时，实际上关于开源还有很多其他事情我在这篇文章中没有谈到。[Artem Zakharchenko](https://twitter.com/kettanaito) 写了一篇很棒的文章 [开源的黑暗面](https://kettanaito.com/blog/the-dark-side-of-open-source)，从不同的角度和观点，我也非常认同。强烈推荐你也读一读。

Reviewer: ```suggestion
另外，还有许多关于开源方面的话题，我在这篇文章里没有提及。[Artem Zakharchenko](https://twitter.com/kettanaito) 写了一篇相当精彩的文章 [开源的黑暗面](https://kettanaito.com/blog/the-dark-side-of-open-source)，他从不同的角度和观点出发，我也非常认同。强烈推荐你也读一读。
```

> ### 感谢

Reviewer: ```suggestion
### 致谢
```

> 最后，我要非常感谢我的女朋友 [Inès](https://www.instagram.com/iiiiiiines__/)，她从一开始就一直支持我，帮助我度过了那些艰难的时刻。没有她的大力支持，我可能今天不会在开源中。

Reviewer: ```suggestion
最后，我要非常感谢我的女朋友 [Inès](https://www.instagram.com/iiiiiiines__/)，她一直支持我，从始至终陪伴我度过每一个艰难的时刻。如果没有她的支持，我的开源之路也许不会走到今天。
```
